curators.md

```diff
@@ -0,0 +1,9 @@
+# Curators
+
+Who's curating which categories of issues?
+These are probably the best people to ask if opening an issue is appropriate.
+
+Again, if in doubt, ask in #elm-community or #elm-dev on [Slack](http://elmlang.herokuapp.com/).
+
+| Category | Description | Curator Name | Preferred Contact Information |
+|----------|-------------|--------------|-------------------------------|
```

Comment:
Category is probably a issue label, description is a succinct description. Perhaps they could be the same thing.